locales/da.json

```diff
@@ -385,7 +385,9 @@
       }
     },
     "aria-required-children": {
-      "pass": "Krævet ARIA-under-elementer er til stede",
+      "pass": {
```

Comment:
I know we normally don't update translations with this sort of change, but I thought it made sense to fix the structure here - with these updates, `npx grunt translate` will preserve the old translations, and without these updates, it'll eat the existing translations and replace them with english.